src/lib/utils.py

```diff
@@ -113,24 +127,27 @@ def extract_appimage_version(basename, data):
             fp.write(data)
 
         os.chmod(appimage_path, 0o755)
-        args = ["bwrap"]
-        for path in ("/usr", "/lib", "/lib64", "/bin", "/proc"):
-            args.extend(["--ro-bind", path, path])
-        args.extend(
-            [
-                "--bind",
-                tmpdir,
-                tmpdir,
-                "--die-with-parent",
-                "--new-session",
-                "--unshare-all",
-                appimage_path,
-                "--appimage-extract",
-            ]
-        )
-        log.debug("$ %s", " ".join(args))
+        if _check_bwrap():
+            args = ["bwrap"]
+            for path in ("/usr", "/lib", "/lib64", "/bin", "/proc"):
+                args.extend(["--ro-bind", path, path])
+            args.extend(
+                [
+                    "--bind",
+                    tmpdir,
+                    tmpdir,
+                    "--die-with-parent",
+                    "--new-session",
+                    "--unshare-all"
+                ]
+             )
+        else:
+            args = []
+            args.extend([appimage_path,"--appimage-extract"])
```

Comment:
You aren't actually putting these two elements into the `bwrap` command any more

Reply:
This is now wrong in the `bwrap` case again :(

Reply:
ah, fixed by later commit